package.json

```diff
@@ -35,21 +35,26 @@
   ],
   "dependencies": {
     "@rdfjs/types": "*",
+    "@shexjs/parser": "1.0.0-alpha.26",
     "@types/async-lock": "^1.4.0",
     "@types/lru-cache": "^5.0.0",
     "@types/mkdirp": "^1.0.0",
+    "@types/semver": "^7.5.7",
     "async-lock": "^1.4.0",
     "componentsjs": "^5.0.0",
+    "jsonld-streaming-parser": "^3.3.0",
     "lru-cache": "^6.0.0",
     "mkdirp": "^1.0.0",
     "rdf-parse": "^2.0.0",
     "rdf-serialize": "^2.0.0",
     "rdf-string": "^1.6.0",
     "rdf-terms": "^1.11.0",
-    "relative-to-absolute-iri": "^1.0.0"
+    "relative-to-absolute-iri": "^1.0.0",
+    "semver": "^7.6.0"
```

Comment:
Where is `semver` used? I tried to look for it but could not find it.

Reply:
I don't know why at some point I had a error telling me that this library was missing but now I don't have this error anymore so I will delete it.